@xen-orchestra/lite/src/components/ui/UiCard.vue

```diff
@@ -1,15 +1,24 @@
 <template>
-  <div class="ui-card">
+  <div class="ui-card" :class="{ error: props.color === 'error' }">
     <slot />
   </div>
 </template>
 
+<script lang="ts" setup>
+const props = defineProps<{
+  color?: "error";
+}>();
+</script>
+
 <style lang="postcss" scoped>
 .ui-card {
   height: fit-content;
   padding: 2.1rem;
   border-radius: 0.8rem;
-  background-color: var(--background-color-primary);
```

Comment:
Why did you removed the card background? It breaks the design.

![Card background](https://user-images.githubusercontent.com/19408/225374162-c4d671a8-cfc9-4fe1-bb51-1236c2b0254e.png)